demo/standalone/Vagrantfile

```diff
@@ -38,7 +38,7 @@ if private_build
   aci_location = "/release/quay.io-rook-rookd-dev.aci"
   aci_options = "--insecure-options=image"
 else
-  aci_location = "quay.io/rook/rookd:latest"
+  aci_location = "quay.io/rook/rookd:master-latest"
```

Comment:
any other instances of this?  i know we still don't have a good story for the zip/archive download, as seen in #616